ethstats/ethstats.go

```diff
@@ -395,10 +394,10 @@ func (s *Service) login(conn *websocket.Conn, sendCh chan *StatsPayload) error {
 		}
 	}
 	auth := &authMsg{
-		ID:      s.node,
+		ID:      s.backend.Address().String(),
```

Comment:
Is this line run by a proxy?  If so, not that the address returned here will be the proxy's etherbase's address.  Not the proxied validator address.

To get the proxied validator address, you need to use `backend.ValidatorAddress`.

Reply:
The line should not be run by a proxy. The ID here is less important anyway because it will be overridden by the address of the account that was used to sign the auth, stats and block informations. It just can not be empty, otherwise celostats will fail.